pallets/runtime/src/bridge.rs

```diff
@@ -177,14 +183,9 @@ decl_event! {
     {
         /// Confirmation of a signer set change.
         ControllerChanged(AccountId),
-        /// Confirmation of minting POLY on Polymesh in return for the locked ERC20 tokens on
+        /// Confirmation of minting POLYX on Polymesh in return for the locked ERC20 tokens on
```

Comment:
Looks like we don't emit ControllerChanged. We should also have an event for AdminChanged.

Reply:
And TimelockChanged ;-)

Reply:
Added.